tests/hpc/ww4_await_pxe_install.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Compute node of warewulf4 cluster
+#     Once controller is up and running, run a VM which is provisioned
+#     by the controller, and run checks.
+# Maintainer: Kernel QE <kernel-qa@suse.de>
+
+use Mojo::Base qw(hpcbase), -signatures;
+use testapi;
+use lockapi;
+use utils;
+use mm_tests;
+use POSIX 'strftime';
+
+sub run ($self) {
+    my $i = 0;
+
+    while ($i < 10) {
+        check_screen('pxe-start');
+        record_info "pxe0", "pxe0";
+        sleep(1);
+        send_key "ctrl-B";
+        sleep(20);
```

Comment:
Direct `sleep` use is usually bad sign and should be avoided. I think multimachine barrier can be inserted before reboot and we can wait for controller.

Reply:
I agree about the sleep. But i consider this a special case. Also look at my reply https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16622#discussion_r1182909512. the whole loop should be reworked in the near future.